en/docs/extended_topics/model_mixed_parallel.md

```diff
@@ -20,47 +20,47 @@ We need to set up a simple multi-layer network first and use this network to dis
 
 ![纯数据并行](imgs/para_consistent_data.png)
 
-In pure data parallelism, we use two devices for training. Because we use **pure data parallelism**. We can see that for each original logical layer, the sample is divided in average to each device. We have complete **training model** in each device. The data after cutting processed by `op`. Finally we combine the data in each GPU and get the full complete data.
+In data parallelism, we use two devices for training. As we use **data parallelism**, we can see that for each original logical layer, the sample is divided in average to each device. We have complete **training model** in each device. The data after splitting processed by `op`. Finally we combine the data in each device and get the complete data.
 
-### Pure model parallelism
+### Model parallelism
 
-In `consistent` view, we can choose pure model parallelism (the configuration details we will talk about it later). The flow diagram is as follows:
+In `consistent` view, we can choose model parallelism (the configuration details we will talk about it later). The flow diagram is as follows:
 
 ![纯模型并行](imgs/para_consistent_model.png)
 
-In pure model parallelism example, we still use two devices for training. In each layer of original logic model is processed by `op  `on **part of model** and **complete data**. Then combine the output and get whole results.
+In model parallelism example, we still use two devices for training. In each layer of original logic model is processed by `op  `on **part of model** and **complete data**. Then combine the output and get whole results.
 
 One thing we need to mention is in above figure. The output from each device on layer 0 **cannot** use as the input in layer 1: Because in model parallelism, in order to complete the operation. We need partial model and **complete** data. To solve this problem, OneFlow use `boxing` mechanism.
 
-`boxing` will count the data in each nodes in distributed training and divide or assemble data properly then send to corresponding GPU. Except the model assembling in model parallelism. The reverse gradient synchronization in data parallelism also will use  `boxing`  to solve problem.
+`boxing` will count the data in each node in distributed training and divide or assemble data properly then send to corresponding GPU. Except the model assembling in model parallelism. The reverse gradient synchronization in data parallelism will also use `boxing`  to solve problem.
 
-The algorithm in `boxing` is complex. But it is open to users. The reason of adding  for adding `boxing` is for keep user from confused. In this article, we only need to remember that OneFlow will automatically solve the data distribution issue.
+The algorithm in `boxing` is complex. But it is transparent to users. The reason of adding `boxing` is to keep user from confused. In this article, we only need to remember that OneFlow will automatically solve the data distribution issue.
```

Comment:
The reason of adding `boxing` is to keep user from confused.
->
 Illustration of boxing is just to prevent users from being confused.

原文的意思会以为是OneFlow中发明Boxing是防止用户迷惑